### ISSUE TITLE
SABnzbd - "Failed to run the package service"

### ISSUE THREAD

**Issue by davco9200** [SynoCommunity/spksrc#4251]
_For new Package Requests, see the guidelines_  

### Setup
_Package Name:_  Sabnzbd
_Package Version:_ 3.1.0-44

_NAS Model:_  DS918+
_NAS Architecture:_ Intel 
_DSM version:_ DSM 6.2.3-25426 Update 2

### Expected behavior
Tell us what should happen

Running the package starts the service

### Actual behavior
Tell us what happens instead

Launching the package fails. 

### Steps to reproduce
Got oPackage Center / Installed
Click Run
Get "Loading" waiting prompt
get the Response "Failed to run the package service"
Click okay. 
Sabnzbd - still says "Run"

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```

Insert the package log here

Sun Nov  1 05:42:01 PST 2020
Starting sabnzbd command env LANG=en_US.UTF-8 /volume1/@appstore/sabnzbd/env/bin/python3 /volume1/@appstore/sabnzbd/share/SABnzbd/SABnzbd.py -f /volume1/@appstore/sabnzbd/var/config.ini --pidfile /volume1/@appstore/sabnzbd/var/sabnzbd.pid -d
Not all required Python modules are available, please check requirements.txt
Missing module: sgmllib
You can read more at: https://sabnzbd.org/python3
If you still experience problems, remove all .pyc files in this folder and subfolders
```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
```

**Comment by gerritjanh** [comment]
I have the exact same problem on mij DS214play.
Tried so solve it by:
uninstalling both sabnzbd and Python3
installing sabnzbd again, this autmatically re-installs Python3.
This didn't solve it, still the same problem.

The search continues. I read here:
https://docs.python.org/2/library/sgmllib.html
that:
The sgmllib module has been removed in Python 3.
???

**Comment by MuadDib007** [comment]
Same on my DS1515+. Before upgrade to 3.1.0 it was working fine, after update Sabnzbd does no longer start with the same error as mentioned above.

**Comment by vanekkendonk** [comment]
Same on DS918+. Is there a easy way to downgrade for the time being?

**Comment by gerritjanh** [comment]
I uninstalled 3.1, than manually installed version 3.0.2-43. 
No problems, just don't hit the Update button.

**Comment by rob040** [comment]
Same on DS920+, I just setup last week and got it working nicely. 
Gone though same steps of de-install sabnzbd, install sabnzbd,  deinstall sabnzbd, de-install python3, install sabnzbd. 
Still "Failed to run the package service"

Logfile sabnzbd.log says:
`Starting sabnzbd command env LANG=en_US.UTF-8 /volume1/@appstore/sabnzbd/env/bin/python3 /volume1/@appstore/sabnzbd/share/SABnzbd/SABnzbd.py -f /volume1/@appstore/sabnzbd/var/config.ini --pidfile /volume1/@appstore/sabnzbd/var/sabnzbd.pid -d`
`Not all required Python modules are available, please check requirements.txt`
**`Missing module: sgmllib`**
`You can read more at: https://sabnzbd.org/python3`
`If you still experience problems, remove all .pyc files in this folder and subfolders`

library sgmllib was already set to deprecated since Python version 2.6 !

@gerritjanh  Question: how to obtain and manual install sabnzbd previous version ?

**Comment by Safihre** [comment]
@ymartin59 I think we need a version bump.

**Comment by gerritjanh** [comment]
@rob040 
you can download the package here:
https://synocommunity.com/package/sabnzbd
Find the right link for the Geminilake processor, download and install manually using the package center.

**Comment by irrg** [comment]
i don't think there's a serious need for more confirmations, but, just in case: also happening on my DS415+.

**Comment by ymartin59** [comment]
I have published fix as 3.1.0-45
